gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -171,12 +173,23 @@ private static void configureErrorProneOptions(
                 @Override
                 public Iterable<String> asArguments() {
                     // Don't apply checks that have been explicitly disabled
-                    Stream<String> errorProneChecks = getSpecificErrorProneChecks(project)
-                            .orElseGet(() -> getNotDisabledErrorproneChecks(
-                                    project, errorProneExtension, javaCompile, maybeSourceSet, errorProneOptions));
-                    return ImmutableList.of(
-                            "-XepPatchChecks:" + Joiner.on(',').join(errorProneChecks.iterator()),
-                            "-XepPatchLocation:IN_PLACE");
+                    Optional<List<String>> specificChecks = getSpecificErrorProneChecks(project);
+                    if (specificChecks.isPresent()) {
+                        List<String> errorProneChecks = specificChecks.get();
+                        return Iterables.concat(
+                                errorProneChecks.stream()
```

Comment:
Could you add an inline comment here describing why this addition is necessary?

Reply:
Done